reports_download.py

```diff
@@ -90,22 +96,26 @@
     report_down = driver.find_element_by_css_selector('body > app-root > div > app-student-solve > div.container > app-solve-question > div > div > div.solution > mat-card > div.result.ng-star-inserted > a.result.mat-elevation-z2.ng-star-inserted')
     while True:
 
-      # Scroll to the botton of the page. 
+      # Scroll to the bottom of the page. 
       driver.execute_script("window.scrollTo(0, document.body.scrollHeight);")
 
-      # Downloading Report
+      # Downloading Report.
       report_down.click()
       time.sleep(0.5)
+
       report_list = glob(os.path.join(download_path,"*.png"))
       if report_list:
         new_folder_path = os.path.join(download_path, "folder_{}".format(num)) 
         os.mkdir(new_folder_path)
         for report in report_list:
           shutil.move(report, new_folder_path)
-        break
+      break
```

Comment:
You sure about this change ?

Reply:
Yeah, otherwise it keeps looping the download button

Reply:
That is how it's supposed to be, because elab messes up sometimes and doesn't downloads the report in a single click and so it makes folders for each question and puts seperate report in them. and then you use image extractor to get a single image from them.

Reply:
I'm aware, but I'm unable to verify that right now because eLab is closed for download. I do remember having an issue with the code being stuck at the same question, and I had resolved the error differently, locally:

```import os
import time

def latestDownloadedFile():
      path = r'Downloads folder file path'
      os.chdir(path)
      files = sorted(os.listdir(os.getcwd()), key=os.path.getmtime)
      newest = files[-1]

      return newest

def checkFile():
  fileends = "crdownload"
  while "crdownload" == fileends:
      time.sleep(1)
      newest_file = latestDownloadedFile()
      if "crdownload" in newest_file:
          return False
      else:
          return True

while checkFile():
  report_down.click()```
  
 This will work only in chrome, but essentially, it checks the download_path (i.e. Downloads folder) every second for the "crdownload" file, and if found, it moves on to the next code snippet.
 
I think I'll change it back to its original position for now, but we can recheck that once eLab is actually responsive, hope that's okay

Reply:
As you can see this is the codeblock that does the same thing 
https://github.com/Aakash-kaushik/elab_report_downloader/blob/6aed8b464af10768e65bf794fa4f37d942d8e4a6/reports_download.py#L99-L105
It downloads reports if it finds them in the folder it moves one into the specific folder for that question.

Reply:
So I think for now let's leave the break that way and we can implement something like your's in future.

Reply:
Alright, works, thanks. Is there anything else? If not, I'll change the placement of the ```break``` statement

Reply:
No, just that break part.